### PR TITLE
Seed

### PR DESCRIPTION
**Pull request by jackbackes**
still rough and untested, so please review and comment.

### CODE REVIEW

File: seed.js [hunk 1]
```diff
@@ -43,6 +47,93 @@ var seedUsers = function () {
...
+  // set composer for each song.
+  return seedSongs.spread( (symphonyNo5) => Promise.all([
+    symphonyNo5.setComposer(composers.beethoven)
+  ]))
+}
+
+let seedOrders = function(){
+  let orders = [testingOrder] = [
+    {
+      time: Date.now(),
```
Comment (geoffbass) [inline seed.js]: can be removed

File: seed.js [hunk 2]
```diff
@@ -43,6 +47,93 @@ var seedUsers = function () {
...
+      time: testingOrder.time
+    }, {
+      include: [
+        //include the correct user
+        {model: User, where: {id: testingOrder.userId}},
+        //include the songs into the order
+        {model: Song, where: {id: [testingOrder.songs[0].songId, testingOrder.songs[1].songId, testingOrder.songs[2].songId]}}
+      ]
+      //add the shipping address
+    } ).createAddress( testingOrder.shippingAddress )
```
Comment (geoffbass) [inline seed.js]: probably needs to be called inside a .then

File: seed.js [hunk 3]
```diff
@@ -17,10 +17,14 @@ name in the environment files.
...
-var User = db.model('user');
-var Promise = require('sequelize').Promise;
+const chalk = require('chalk');
+const db = require('./server/db');
+const User = db.model('user');
+const Order = db.model('order');
+const Review = db.model('review');
+const Song = db.model('song');
+const Composer = db.model('composer');
+const Promise = require('sequelize').Promise;
```
Comment (jonperrelle) [inline seed.js:31]: need to require the Genre Table

File: seed.js [hunk 4]
```diff
@@ -43,6 +47,93 @@ var seedUsers = function () {
...
+  //populate this object with composer data. set an object title so that it can be used in the "setComposer" command below.
+  let composers = {
+    beethoven: {
+      firstName: "Ludwig",
+      lastName: "Van Beethoven"
+    }
+  }
+
+  // set composer for each song.
+  return seedSongs.spread( (symphonyNo5) => Promise.all([
```
Comment (jonperrelle) [inline seed.js]: does this need to be seedSongs().spread?

File: seed.js [hunk 5]
```diff
@@ -43,6 +47,93 @@ var seedUsers = function () {
...
+    symphonyNo5.setComposer(composers.beethoven)
+  ]))
+}
```
Comment (jonperrelle) [inline seed.js:108]: Does the setComposer method seed the composers model, or do we need to create Composer?

File: seed.js (hunk 4 above)
Comment (geoffbass) [inline seed.js]: I think that's just so we can add more songs to the seed file later without changing the code. Is that right @thejohnbackes ?

File: seed.js [hunk 6]
```diff
@@ -43,6 +47,93 @@ var seedUsers = function () {
...
+        city: "MoonBase",
+        state: "TX",
+        zipCode: 75252
+      }
+    }]
+  return seedUsers.spread( (testing, obama) => [
+    testing.createOrder( {
+      //add the time of the order
+      time: testingOrder.time
+    }, {
```
Comment (jonperrelle) [inline seed.js:132]: Remove time

File: seed.js [hunk 7]
```diff
@@ -43,6 +47,93 @@ var seedUsers = function () {
...
+    testing.createReview( {
+      rating: testingReview.rating,
+      description: testingReview.description
+    } )
+  ])
+};
+
 db.sync({ force: true })
     .then(function () {
         return seedUsers();
```
Comment (jonperrelle) [inline seed.js]: Need to seed genre model

File: seed.js (hunk 4 above)
Comment (jackbackes) [inline seed.js]: good catch, @jonperrelle

File: seed.js (hunk 5 above)
Comment (jackbackes) [inline seed.js:108]: there is a createComposer association method. I'll use that.

**Pull request closed without merging**